requirements.txt

```diff
@@ -1,15 +1,16 @@
 alembic==1.7.4
-canonicalwebteam.flask-base==0.9.1
+canonicalwebteam.flask-base==0.9.2
```

Comment:
I think `canonicalwebteam.flask-base==0.9.2` should be in the first place? The Python version used will be determined by the first package.

Reply:
Why would the order of the packages affect the python version?

Reply:
[Here is an example](https://github.com/canonical-web-and-design/dqlite.io/pull/127)

Reply:
Interesting, thanks @carkod I will put it at the beginning of the file